ibis/impala/tests/test_client.py

```diff
@@ -25,6 +25,21 @@ def db(con, test_data_db):
     return con.database(test_data_db)
 
 
+def test_kerberos_deps_installed(env, test_data_db):
+    with pytest.raises(AttributeError):
```

Comment:
Can you have inside the `with` only the line that can raise the exceptiion, and not the whole block please.